src/pages/components/DashboardHeader/DashboardHeader.tsx

```diff
@@ -0,0 +1,83 @@
+import { ArrowBackIcon, ChevronRightIcon, CopyIcon } from '@chakra-ui/icons';
+import {
+  Box,
+  Input,
+  InputGroup,
+  InputRightElement,
+  Text,
+  Tooltip,
+  useBreakpointValue,
+  useToast,
+} from '@chakra-ui/react';
+import { useRouter } from 'next/router';
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface DashboardHeaderProps {
+  pageTitle: string;
+}
+
+const DashboardHeader: React.FC<DashboardHeaderProps> = function Dashboard({ pageTitle }) {
+  const router = useRouter();
+  const { t } = useTranslation('dashboard');
+  const [apiKey, setApiKey] = useState('');
+  const toast = useToast();
+  const ml = useBreakpointValue({ base: 10, lg: 270 });
+  const pageHeaderDisplay = useBreakpointValue({ base: 'block', lg: 'flex' });
+
+  const handleGoBack = () => {
+    router.back();
+  };
+
+  const handleCopyApiKey = () => {
+    setApiKey('example-api-key');
+    navigator.clipboard.writeText(apiKey);
+    toast({
+      title: t('API key copied'),
+      description: t('Your API key has been copied to your clipboard.'),
+      status: 'success',
+      duration: 5000,
+      isClosable: true,
+    });
+  };
+
+  return (
+    <>
```

Comment:
I would wrap this in `Box` rather than a React fragment so these elements are wrapped in a DOM element